### PR TITLE
tease Camp Release on landing page

### PR DESCRIPTION
**Pull request by gr2m**
Okay if we tease the new Hoodie release on the landing page? I prepare something, not perfect but works I think. I’d expect an increased interest in the coming days & weeks, so would be nice to put a link to the new release in there

screen shot 2016-05-16 at 16 21 02

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,19 +32,17 @@
...
-                    <p>
-                        Learn in <a href="/intro#get-started">these tutorials</a> how to install Hoodie and get started.
-                    </p>
-                </section>
-            </div>
+          <div class="grid-3">
+              <section class="teaser-index-section">
+                  <h3>Hoodie Camp Release</h3>
+                  <p>
+                      The new Hoodie is around the corner, you can <strong><a href="http://github.com/hoodiehq/hoodie/" target="_blank">give it a try today</a></strong>!
```
Comment (NickColley) [inline index.html:39]: target="_blank" is an accessibility issue as assisted tech users will lose context

(it's also a security issue but not really a problem here I guess)

Comment (gr2m) [inline index.html:39]: I agree with both, but we do the same with all the other links since a long time so went with the flow

File: index.html [hunk 2]
```diff
@@ -32,19 +32,17 @@
...
+                  </p>
+                  <p>
+                    <a href="http://github.com/hoodiehq/hoodie/" target="_blank" style="background: transparent;"><img src="/dist1/content_img/index/hoodie-camp-transparent.png" alt="Hoodie Camp" style="border: 0;"/></a>
```
Comment (NickColley) [inline index.html:42]: are the inline styles required?

the alt tag could be more descriptive

**Comment by NickColley** [comment]
Looks cute :) Added a few notes

File: index.html (hunk 2 above)
Comment (gr2m) [inline index.html:42]: yeah I tried to use the CSS but I think our whole setup is borked and I don’t get how it works :( I’m not sure about the alt text, it’s not really adding any valuable content, we might as well set `alt=""` to have it ignored, as it’s just decorative?

**Comment by jennwrites** [comment]
LGTM

**Pull request merged**